### ISSUE TITLE
When running on linux I get told I need FUSE

### ISSUE THREAD

**Issue by MuffinzCat** [DevilXD/TwitchDropsMiner#476]
As the title suggests, I'm encountering difficulties running an app on Linux; it fails to open. When I attempt to execute it via the console, I receive the following error message:
```
dlopen(): error loading libfuse.so.2

AppImages require FUSE to run.
You might be able to extract the contents of this AppImage
if you run it with the --appimage-extract option.
See https://github.com/AppImage/AppImageKit/wiki/FUSE
for more information.
```
However, when I try to install fuse using sudo apt install fuse, it prompts the removal of numerous packages:
```
  fuse3 gnome-remote-desktop gnome-shell-extension-desktop-icons-ng gvfs-fuse
  nautilus shotwell ubuntu-desktop-minimal ubuntu-session xdg-desktop-portal
  xdg-desktop-portal-gnome xdg-desktop-portal-gtk
```
These appear to be many packages that I would prefer to retain, such as ubuntu-desktop.

**Comment by MuffinzCat** [comment]
I attempted to locate the PyInstaller file as suggested by the readme, but it appears to be missing.

**Comment by guihkx** [comment]
I'm not sure why you closed this, but you probably just need to install `libfuse2`.

Alternatively, you can set this environment variable before running the AppImage: `APPIMAGE_EXTRACT_AND_RUN=1`.

**Comment by MuffinzCat** [comment]
As I said in the closing comment installing libfuse2 worked
________________________________
Van: Guilherme Silva ***@***.***>
Verzonden: dinsdag 21 mei 2024 19:20
Aan: DevilXD/TwitchDropsMiner ***@***.***>
CC: Muffin ***@***.***>; State change ***@***.***>
Onderwerp: Re: [DevilXD/TwitchDropsMiner] When running on linux I get told I need FUSE (Issue #476)


I'm not sure why you closed this, but you probably just need to install libfuse2.

Alternatively, you can set this environment variable before running the AppImage: APPIMAGE_EXTRACT_AND_RUN=1.

**Comment by guihkx** [comment]
>As I said in the closing comment installing libfuse2 worked

You might have posted elsewhere, because there's no such post from you here... lol

**Comment by MuffinzCat** [comment]
right, yeah I closed it since I'm using a fork and not this version, the version here just doesn't work in general for me

**Comment by MuffinzCat** [comment]
> right, yeah I closed it since I'm using a fork and not this version, the version here just doesn't work in general for me

I replied by email, so I didn't see it wasnt the correct repo, mb
